### DISCUSSION TITLE
[destination-elasticsearch] add support to ES 6.1.4 version

### DISCUSSION THREAD

**Discussion by hli191** [airbytehq/airbyte#33164] [Connector Ideas and Features]
### What area the feature impact?

Connectors

### Relevant Information

For now only ES above 7+ support by destination。In our environment want move data from Source to destination (ES 6.1.4).

**Comment by marcosmarxm** [comment]
Thanks for requesting the feature @hli191 at the moment ElasticSearch is not in the connector team roadmap but you're free to edit and make the yourself :)

[a comment by an automated account is not shown]
